include/common/configure.h

```diff
@@ -115,8 +116,9 @@ class RuntimeConfigure {
   RuntimeConfigure(const RuntimeConfigure &RHS) noexcept
       : MaxMemPage(RHS.MaxMemPage.load(std::memory_order_relaxed)),
         EnableJIT(RHS.EnableJIT.load(std::memory_order_relaxed)),
+        EnableCoredump(RHS.EnableCoredump.load(std::memory_order_relaxed)),
         ForceInterpreter(RHS.ForceInterpreter.load(std::memory_order_relaxed)),
-        AllowAFUNIX(RHS.AllowAFUNIX.load(std::memory_order_relaxed)) {}
```

Comment:
Please apply clang-format and clang-tidy

Reply:
Sure thing. Seems like clang-format wasn't running correctly in my IDE. Apologies